index.html

```diff
@@ -2,8 +2,85 @@
 <html lang="en">
 <head>
     <title>James Cullen Portfolio</title>
+    <link rel="stylesheet" type="text/css" href= "styles.css"/>
 </head>
-<body>
+<body class="style">
+    <header>
+        <nav class="container">
+            <div>
+                <ul>
+                    <li class="nav-buttons">
+                        <a href="#contact">contact</a>
+                    </li>
+                    <li class="nav-buttons">
+                        <a href="#portfolio">portfolio</a>
+                    </li>
+                    <li class="nav-buttons">
+                        <a href="#about">about</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
 
+        <main class="container">
+            <section>
+                <article>
+                    <h1 id="about">about</h1>
```

Comment:
Multiple use of 'about' ID - maybe class is a better selector in this instance? Lines 29, 37 and 45.

Reply:
using ID in this instance as a link / bookmark to navigate the site on a single page - I have moved the id="about" to the relevant section selector